ngx-fudis/projects/ngx-fudis/src/lib/components/link/link.component.ts

```diff
@@ -32,7 +32,7 @@ import { FudisFocusService } from '../../services/focus/focus.service';
  * <fudis-link
  *     [href]="https://www.example.com"
  *     [isExternalLink]="true"
- * 	color="default">
+ * 	[color]="'default'">
  * </fudis-link>
```

Comment:
I think we should aim to remove these *.ts examples, and focus on keeping Storybook documentation updated.

Reply:
Removed those examples from typescript and updated documentation by adding readme.mdx